zzz_generated/all-clusters-app/zap-generated/access.h

```diff
@@ -0,0 +1,364 @@
+/*
+ *
+ *    Copyright (c) 2022 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+// THIS FILE IS GENERATED BY ZAP
+
+// Prevent multiple inclusion
+#pragma once
+
+#define GENERATED_ACCESS_PRIVILEGE__VIEW (0)
+#define GENERATED_ACCESS_PRIVILEGE__OPERATE (1)
+#define GENERATED_ACCESS_PRIVILEGE__MANAGE (2)
+#define GENERATED_ACCESS_PRIVILEGE__ADMINISTER (3)
+
+////////////////////////////////////////////////////////////////////////////////
+
+// clang-format off
+#define GENERATED_ACCESS_READ_ATTRIBUTE__CLUSTER { \
```

Comment:
could we generate a comment here detailing what the layout of this array is?

In reviewing code, the `31` and `257` and such are not obvious.

Reply:
I added a comment, it should be clearer now.